### DISCUSSION TITLE
Help running tailscale LXC script

### DISCUSSION THREAD

**Discussion by schlangens** [tteck/Proxmox#2267] [Help Needed] [answered]
I would like to have someone test the LXC script to install Tailscale. I am using the shell and have never had an issue installing other scripts. When I execute the command, the system sits, and I am given the prompt back to enter another command. There is no error that I can see. I tried the ``set -x`` in hopes that I would be given an error, and I was not. The last thread I came across was in October, and I can't find the right direction to solve this. I am on Proxmox V8, and my last reboot was yesterday.

**Comment by tteck** [comment]
The script runs without any problems from this end. 

```
  ______      _ __                __
 /_  __/___ _(_) /_____________ _/ /__
  / / / __ `/ / / ___/ ___/ __ `/ / _ \
 / / / /_/ / / (__  ) /__/ /_/ / /  __/
/_/  \__,_/_/_/____/\___/\__,_/_/\___/

Installing Tailscale...
Installed Tailscale
 ✔ Completed Successfully!
 Reboot 200 LXC to apply the changes, then run tailscale up in the LXC console
root@server:~# 
```

**Comment by tteck** [comment]
What is the output from executing this command `pct list | awk 'NR>1'`?

**Comment by schlangens** [comment]
Thank you for your timely response @tteck. I get the following from the previous command:
```
root@ss-prox:~# pct list | awk 'NR>1'
100        running                 pihole              
101        running                 ubuntu              
102        running                 qbittorrent         
103        running                 cloudflared         
105        running                 dashy               
106        running                 whoogle  
```

**Reply by tteck** [reply]
Does a dialog box pop up asking you to choose one of those?

**Reply by schlangens** [reply]
No

**Reply by tteck** [reply]
what's the output from `ping github.com`?

**Reply by tteck** [reply, marked as answer]
I'm going to guess that you have DNS issues?

**Reply by schlangens** [reply]
As expected, I tried running that and was getting a weird DNS issue. Thanks for the quick reply. I think I got it from here.
